infra/helper.py

```diff
@@ -959,9 +961,12 @@ def shell(args):
       'FUZZING_ENGINE=' + args.engine,
       'SANITIZER=' + args.sanitizer,
       'ARCHITECTURE=' + args.architecture,
-      'FUZZING_LANGUAGE=' + _get_project_language(args.project_name),
   ]
 
+  lang = _get_project_language(args.project_name)
```

Comment:
Can you please make this to skip like this. it shouldnt be needed outside of this one gdb usecase.
if args.project_name != 'base-runner-debug':
  env.append('FUZZING_LANGUAGE='+_get_project_language(args.project_name))

Reply:
Easy enough. I've already pushed an updated branch. :)